File: VB/DevExpress.AI.Samples.Blazor/Data/Issue.vb
Namespace DevExpress.AI.Samples.Blazor.Data

    Public Enum IssueType
        Request
        Bug
    End Enum

    Public Enum IssueStatus
        [New]
        Postponed
        Fixed
        Rejected
    End Enum

    Public Enum IssuePriority
        Low
        Medium
        High
    End Enum

    Public Partial Class Issue

        Public Property ID As Long

        Public Property Name As String

        Public Property Type As IssueType

        Public Property ProjectID As Nullable(Of Long)

        Public Property Priority As IssuePriority

        Public Property Status As IssueStatus

        Public Property CreatorID As Nullable(Of Long)

        Public Property CreatedDate As Nullable(Of System.DateTime)

        Public Property OwnerID As Nullable(Of Long)

        Public Property ModifiedDate As Nullable(Of System.DateTime)

        Public Property FixedDate As Nullable(Of System.DateTime)

        Public Property Description As String

        Public Property Resolution As String
    End Class
End Namespace

File: CS/DevExpress.AI.Samples.Blazor/Data/Issue.cs
﻿namespace DevExpress.AI.Samples.Blazor.Data {
    public enum IssueType { Request, Bug }
    public enum IssueStatus { New, Postponed, Fixed, Rejected }
    public enum IssuePriority { Low, Medium, High }
    public partial class Issue {
        public long ID { get; set; }
        public string Name { get; set; }
        public IssueType Type { get; set; }
        public Nullable<long> ProjectID { get; set; }
        public IssuePriority Priority { get; set; }
        public IssueStatus Status { get; set; }
        public Nullable<long> CreatorID { get; set; }
        public Nullable<System.DateTime> CreatedDate { get; set; }
        public Nullable<long> OwnerID { get; set; }
        public Nullable<System.DateTime> ModifiedDate { get; set; }
        public Nullable<System.DateTime> FixedDate { get; set; }
        public string Description { get; set; }
        public string Resolution { get; set; }
    }
}